shub/image/build.py

```diff
@@ -67,7 +67,7 @@ def build_cmd(target, version, skip_tests):
     click.echo("The image {} build is completed.".format(image_name))
     # Test the image content after building it
     if not skip_tests:
-        test_mod.test_cmd(target, version)
+        test_cmd.test_cmd(target, version)
```

Comment:
Do you think that now it looks better for some reasons? Just out of curiosity, I'm trying to understand motivation as I'd rename it rather to full `test_module`. I was always a bit confused with naming like `from datetime import datetime` 🙂